ArcadiaParties-SPA/src/app/content/content.component.ts

```diff
@@ -18,12 +18,14 @@ import { DepartmentState } from '../department/reducers/reducer';
 export class ContentComponent implements OnDestroy {
   private mergedObservableSubscription: Subscription;
 
-  currentDepartmentId$ = this.store.pipe(select(selectCurrentDepartment));
+  currentDepartmentId$ = this.store.pipe(select(selectCurrentDepartmentId));
 
   constructor(private store: Store<DepartmentState>, route: ActivatedRoute) {
     store.dispatch(changeTitleAction({ title: 'Arcadia Parties' }));
 
-    const routeParams$ = route.params.pipe(filter(params => params.departmentId));
+    const routeParams$ = route.params
+      .pipe(filter(params => params.departmentId))
+      .pipe(map(params => +params.departmentId));
```

Comment:
I said; "Like for `currentUserDepartment$`" :) In single pipe, not in different.

Reply:
Sorry, I hurried. Already done.